docs/reference-guides/data/data-core.md

```diff
@@ -317,6 +317,18 @@ _Returns_
 
 -   `any`: The entity record's save error.
 
+### getNavigationFallback
```

Comment:
This seems to return an Id, should it be `getNavigationFallbackId` or something like that?

Reply:
✅